[BUG] Build html Documentation requires plantuml

### Description / Steps to reproduce the issue

writing output... [ 59%] debugging/mte .. guides/multiple_nsh_sessions           
Warning, treated as error:
/home/alan/nuttxspace/nuttx/Documentation/guides/port_bootsequence.rst:55:plantuml command 'plantuml' cannot be run
make: *** [Makefile:52: html] Error 2


### On which OS does this issue occur?

[OS: Linux]

### What is the version of your OS?

Ubuntu

### NuttX Version

mainline

### Issue Architecture

[Arch: all]

### Issue Area

[Area: Other]

### Host information

_No response_

### Verification

- [x] I have verified before submitting the report.

--- a/Documentation/contributing/documentation.rst
+++ b/Documentation/contributing/documentation.rst
@@ -20,7 +20,7 @@ To render the Documentation locally, you should clone the NuttX main repository 
      project `site <https://github.com/pyenv/pyenv#installation>`_.
 
     .. code-block:: console
-      
+
       $ pip3 install pipenv
       $ cd Documentation/
       $ # install the dependencies into a virtual environment
@@ -28,7 +28,15 @@ To render the Documentation locally, you should clone the NuttX main repository 
       $ # activate the virtual environment
       $ pipenv shell
 
-  2. Build documentation:
+  2. `Install the PlantUML tool <https://www.javiljoen.net/n/installing-plantuml.html>`_
+     (used to render UML diagrams) and ensure it is on your ``PATH``.
+
+     .. code:: console
+
+        $ sudo apt install plantuml
+        $ plantuml -version
+
+  3. Build documentation:
 
     .. code-block:: console
 

